addons/stock/product.py

```diff
@@ -359,8 +359,17 @@ def _product_available(self, cr, uid, ids, field_names=None, arg=False, context=
             if f == 'outgoing_qty':
                 c.update({ 'states': ('confirmed','waiting','assigned'), 'what': ('out',) })
             stock = self.get_product_available(cr, uid, ids, context=c)
+            mto_ids = []
+            check = 'procure_method' in self._all_columns
```

Comment:
In 7.0, several modules add the field 'procure_method', this is why we use a check (no way to overwrite _product_available in one module).